index.html

```diff
@@ -95,7 +95,7 @@ <h2>Multi-Post Stories</h2>
 
             <div class="showcase" id="showcase">
                 <div class="showcase-image">
-                   <img src="images/Img Placeholder (3).png" alt="">
+                   <img src="images/Img Placeholder (3).png" alt="project image">
                 </div>
 
                 <div class="showcase-detail">
```

Comment:
- [ ] the project in your recent works sections should be loaded dynamically using JS code, right now the hardcoded HTML protects are still displayed on the webpage and that's probably the reason the pop-up isn't working either.